docs/release-process.md

```diff
@@ -70,13 +79,9 @@ so this command would update Vitess Operator to build against Vitess v9.0.0:
 go get vitess.io/vitess@daa60859822ff85ce18e2d10c61a27b7797ec6b8
 ```
 
-### Release
-
-After the PR from the prepare phase is merged, make sure your local git dir is
-up-to-date with HEAD, and then create a temporary release branch on top of the long-term release branch, for instance:
-
-```
-git checkout -b new-release-2.7.5 origin/release-2.7
+Following this cleanup the `go.sum` file by:
+```sh
+go mod tidy
 ```
 
 And then, create the tag using the following command, note that you will need to replace the placeholder strings:
```

Comment:
Let's add a sub heading here too, for instance:

```
#### Do-Release
```

Reply:
Fixed